src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Project Duke CS2103
+ * Done by Hong Jin.
+ */
+package duke;
+
+import duke.command.*;
```

Comment:
Maybe you can import each class you need instead of using *?

Reply:
oh yes, thanks for pointing that out :)